pandas_vet/__init__.py

```diff
@@ -86,6 +91,52 @@ def check_for_notnull(node: ast.Call) -> List:
         return [PD004(node.lineno, node.col_offset)]
     return []
 
+def check_for_arithmetic_methods(node: ast.Call) -> List:
+    """
+    Check AST for occurence of explicit arithmetic methods.  
+
+    Error/warning message to recommend use of binary arithmetic operators instead.
+    """
+    arithmetic_methods = [
+        'add',
+        'sub', 'subtract',
+        'mul', 'multiply',
+        'div', 'divide', 'truediv',
+        'pow',
+        'floordiv',
+        'mod',
+        ]
+    arithmetic_operators = [
+        '+',
+        '-',
+        '*',
+        '/',
+        '**',
+        '//',
+        '%',
+        ]
+
+    errors = []
+    if isinstance(node.func, ast.Attribute) and node.func.attr in arithmetic_methods:
```

Comment:
Match new non-append pattern:

```suggestion
    if isinstance(node.func, ast.Attribute) and node.func.attr in arithmetic_methods:
        return [PD005(node.lineno, node.col_offset)]
    return []
```